uint32 startingCol
uint32 endingCol
uint32 startingRow
uint32 endingRow
uint32 noOfPixels
uint32 max_disparity
uint32 noOfPoints
point2[] points
uint8 which_class